test/e2e/common/setup.go

```diff
@@ -56,7 +56,7 @@ var (
 	rootDir       = filepath.Join(filepath.Dir(file), "../../..")
 
 	testHub = env.GetString("HUB", "quay.io/maistra-dev")
-	testTag = env.GetString("TAG", "latest")
+	testTag = env.GetString("TAG", "test")
```

Comment:
Now I just wonder does it make sense to now make these values constants? They are hard-coded in `kind_provisioner.sh`, so why take them from the environment here?

Reply:
> They are hard-coded in kind_provisioner.sh, so why take them from the environment here?

Because you can run `go test -tags=integ -run TestTraffic ./test/e2e/k8s ...`. However, it was incorrect and I pushed fixed version with new environment variable.